ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,59 @@
+import { fizzbuzz } from "./fizzbuzz";
 
 describe("fizzbuzz", () => {
+  it("should return string", () => {
+    expect(typeof fizzbuzz(5)).toBe("string");
+  });
 
+  it.each([3, 6, 9, 12])(
+    "should return fizz for (%i) which is multiplication of 3",
+    (number) => {
+      expect(fizzbuzz(number)).toBe("fizz");
+    }
+  );
 
+  it.each([5, 10, 20])(
+    "should return fizz for (%i) which is multiplication of 5",
+    (number) => {
+      expect(fizzbuzz(number)).toBe("buzz");
+    }
+  );
+
+  it("should return fizzbuzz for 15", () => {
```

Comment:
I think this test case is not fulfilling the requirements which state: "15 returns "FizzBuzz" but also 45 returns "FizzBuzz"". So this test case can be called "Returns FizzBuzz for multiples of 15"